### ISSUE TITLE
Fortune with default NeDB adapter generates invalid IDs with slashes

### ISSUE THREAD

**Issue by unindented** [fortunejs/fortune#112]
If I post a user to my endpoint without specifying an ID:

```
{
  "@graph": [{
    "@type": "user",
    "firstName": "John",
    "lastName": "Doe"
  }]
}
```

The user will be assigned a random ID:

```
{
  "@graph": [{
      "@type": "user",
      "@id": "/users/fckuk1hEVbVYr%2FYs91Kb",
      "id": "fckuk1hEVbVYr/Ys91Kb",
      "firstName": "John",
      "lastName": "Doe"
    }]
}
```

But if I try to GET by that ID, the server will fail:

```
$ curl http://localhost:8001/api/users/fckuk1hEVbVYr%2FYs91Kb
{
  "@error": {
    "name": "NotFoundError",
    "message": "The field \"Ys91Kb\" is not a link on the type \"user\"."
  }
}
```

**Comment by 0x8890** [comment]
Looks like it's a bug regarding URI decoding, will investigate a fix.

**Comment by 0x8890** [comment]
Fixed in commit https://github.com/fortunejs/fortune/commit/b1a9fa158aa3fb1468affee6538c4d4d6f1f56e6, published as `1.0.0-rc.7`.

**Comment by unindented** [comment]
Awesome. Thank you!
